scripts/zkai_d128_full_block_accumulator_backend_gate.py

```
@@ -0,0 +1,987 @@
+    if extra_keys:
+        raise D128FullBlockAccumulatorBackendError(f"payload unexpected keys: {extra_keys}")
+    _source, _artifact, _handle, expected_summary = _validate_common_payload(payload)
+    case_count, rejected = _validate_case_metadata(payload)
+    if rejected != case_count:
+        raise D128FullBlockAccumulatorBackendError("not all full-block accumulator backend mutations rejected")
+    expected_summary["mutation_cases"] = case_count
+    expected_summary["mutations_rejected"] = rejected
+    expect_equal(payload.get("summary"), expected_summary, "summary")
+
+
+def classify_error(error: Exception) -> str:
```

Ruff (0.15.12)

[warning] Too many return statements (9 > 6)

(PLR0911)